.github/workflows/tests.yaml

```diff
@@ -42,6 +40,24 @@ jobs:
           environment-file: conda-store-server/environment-macos-dev.yaml
           auto-activate-base: false
 
+      - name: Set up Python (Windows)
+        if: matrix.os == 'windows-latest'
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          activate-environment: conda-store-server-dev
+          environment-file: conda-store-server/environment-windows-dev.yaml
+          auto-activate-base: false
+
+      # This fixes a "DLL not found" issue importing ctypes from the hatch env
+      - name: Reinstall Python 3.10 on Windows runner
+        uses: nick-fields/retry@v2.8.3
+        with:
+          timeout_minutes: 9999
```

Comment:
FYI this was copied from somewhere else https://github.com/linz/emergency-management-tools/commit/d1d00931544f4a71ec1f7865041b16fd13d907a6

Reply:
Here are the docs https://github.com/nick-fields/retry. I guess the idea here is that it fails after doing max_attempts rather than timeouting, which might be flaky. I think it's fine to keep.